bindinfo/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+options:
+  no_parent_owners: true
```

Comment:
Just for `approvers` here since the field is already set up.
if `reviewers` field is setup, the reviewers will also not inherit from parent owners.

> We hacked the prow instance to support this feature.